pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/NonPersistentPublisherStats.java

```diff
@@ -31,7 +29,9 @@
     public double msgDropRate;
 
     public NonPersistentPublisherStats add(NonPersistentPublisherStats stats) {
-        checkNotNull(stats);
```

Comment:
there is Objects.requireNonNull in the JDK, we replace Preconditions with Objects

Reply:
thanks for your suggestion, applied with https://github.com/apache/pulsar/pull/10697/commits/4b123ae39b32151bc6349a51037469184fe82f96